packages/kbn-react-field/src/field_icon/field_icon.tsx

```diff
@@ -94,11 +94,11 @@ export function FieldIcon({
   return (
     <EuiToken
       {...token}
-      className={classNames('kbnFieldIcon', className)}
```

Comment:
The Data Discovery team was notified due to changes to this file, but it doesn't look like anything changed here other than the order of props. Is this intentional or just leftover from temporary changes?

Reply:
@davismcphee You're right the change that triggered ownership
review was reverted. Apologies for the ping.